CONTRIBUTING.rst

```diff
@@ -49,14 +49,23 @@ To update to the current master@HEAD revision, run:
 
    nix-shell --run 'niv update tahoe-lafs-master --branch master'
 
+We intend for these updates to be performed periodically.
+At the moment, they must be performed manually.
+It might be worthwhile to `automate this process <https://github.com/PrivateStorageio/ZKAPAuthorizer/issues/287>` in the future.
+
 It is also possible to pass ``pull/<pr-number>/head`` to test against a specific PR.
-When feasible we also test against a released version of Tahoe-LAFS.
+
+The Nix package depends on a version of Tahoe-LAFS determined by the "tahoe-lafs" niv entry.
```

Comment:
I know that "nix packaging" == "what is deployed on storage nodes", but it might make sense to be explicit about this.

Maybe something like "The package deployed on our storage node depends on the version of tahoe indicated by the `tahoe-lafs` niv entry..."?

Reply:
Maybe it would also make sense to have this section first, given that it is the one that is consumed outside of CI.